rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMNodeEvent.java

```diff
@@ -214,10 +219,10 @@ public boolean equals(Object obj) {
         if (!(obj instanceof RMNodeEvent))
             return false;
         RMNodeEvent other = (RMNodeEvent) obj;
-        if (nodeUrl == null) {
-            if (other.nodeUrl != null)
+        if (getKey() == null) {
+            if (other.getKey() != null)
                 return false;
-        } else if (!nodeUrl.equals(other.nodeUrl))
+        } else if (!getKey().equals(other.getKey()))
             return false;
         return true;
     }
```

Comment:
Create an utile method for comparing key instead of rewriting the same code twice.

Reply:
getKey() returns nodeUrl.
I use getKey() somewhere else, here i just incapsulate key decision in one method